htdocs/core/modules/modHRM.class.php

```diff
@@ -249,6 +249,22 @@ public function __construct($db)
 		$this->rights[$r][4] = 'compare_advance';
 		$this->rights[$r][5] = 'read'; // In php code, permission will be checked by test if ($user->rights->hrm->compare_advance->read)
 		$r++;
+
+		// Read employee
+		$this->rights[$r][0] = 4031; // Permission id (must not be already used)
+		$this->rights[$r][1] = 'Read employee'; // Permission label
```

Comment:
An employee is a user with object. 
And we already have a permission to read users and a permission for each type of object (leave, expense report, salary, etc...).
Adding such a permission create a lot of business rule conflicts into the code and on some security checks.
i think the permission you want to add is not to define if we can see an employee or not but to decide if we can see some kind of information (like personal information) ?

Reply:
These two new rights is about the personal informations fields only.

Reply:
So can you please rename the label of permission so things are clear. They were not for me and i have to go into the code to understand the permission was to view/read personal information. But end users are not able to do this analyze so having a better label of permission will avoid interrogation.